atom/browser/api/atom_api_app.cc

```diff
@@ -34,8 +34,8 @@
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/icon_manager.h"
 #include "chrome/common/chrome_paths.h"
-#include "content/browser/gpu/compositor_util.h"
-#include "content/browser/gpu/gpu_data_manager_impl.h"
+#include "content/browser/gpu/compositor_util.h"        // nogncheck
+#include "content/browser/gpu/gpu_data_manager_impl.h"  // nogncheck
```

Comment:
can we have additional comments for nogncheck'd files re: why we can't just have the correct dependencies?

Reply:
er nvm, you commented on this in the PR description.